File: javascript/darts/darts.js
export const score = (x, y) => {
  const r = Math.sqrt(Math.pow(x, 2) + Math.pow(y, 2));
  if (r > 10) return 0;
  else if (r > 5) return 1;
  else if (r > 1) return 5;
  else return 10;
};

File: go/darts/darts.go
package darts

import "math"

func Score(x, y float64) int {
	radius := math.Sqrt(math.Pow(x, 2.0) + math.Pow(y, 2.0))
	if radius > 10 {
		return 0
	} else if radius > 5 {
		return 1
	} else if radius > 1 {
		return 5
	} else {
		return 10
	}
}
